tests/e2e/healthcheck/healthcheck_test.go

```diff
@@ -249,41 +250,6 @@ func testAppHealthCheckProtocol(t *testing.T, protocol string) {
 			wg.Wait()
 		})
 
-		t.Run("after delay, counters aren't increasing", func(t *testing.T) {
```

Comment:
Wait! Why is this test deleted?

Reply:
Ping @1046102779

Reply:
@1046102779 any reason why this test is removed?

Reply:
@1046102779 ping